wrappers/wasm/BarcodeReader.cpp

```diff
@@ -96,8 +97,21 @@ EMSCRIPTEN_BINDINGS(BarcodeReader)
 	        .field("format", &ReadResult::format)
 	        .field("text", &ReadResult::text)
 	        .field("error", &ReadResult::error)
+            .field("position", &ReadResult::position)
 	        ;
 
+    value_object<ZXing::PointI>("Point")
+	        .field("x", &ZXing::PointI::x)
+	        .field("y", &ZXing::PointI::y)
+	        ;
+
+    value_object<ZXing::Position>("Position")
+            .field("topLeft", emscripten::index<0>())
+	        .field("topRight", emscripten::index<1>())
+	        .field("bottomLeft", emscripten::index<2>())
+            .field("bottomRight", emscripten::index<3>())
+            ;
+
```

Comment:
You can do this this way (I guess) but be careful of the order: the points are stored in clockwise order. Also, please make sure the whitespace style is consistent with the rest (use tabs with indent level 4 as leading whitespace).